code/controllers/subsystem/mapping.dm

```diff
@@ -0,0 +1,37 @@
+SUBSYSTEM_DEF(mapping)
+   	name = "Mapping"
+   	init_order = 12
```

Comment:
I think you should be using a define for this (see the other subsystems)

Also, `INIT_ORDER_JOBS` is already defined as 12, not sure if that'll break anything but it's probably best to select a different value

Reply:
Noted, will work on these when I am home from starbucks